packages/calcite-components/src/utils/iconScaleAdjuster.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { adjustIconScale } from "./iconScaleAdjuster";
```

Comment:
Could you rename the util module to `component.ts` and the exported function to the following?

```ts
getIconScale(componentScale: Scale): Extract<Scale, "s" | "m">;
```